transports/dns/Cargo.toml

```diff
@@ -29,12 +29,12 @@ async-std-crate = { package = "async-std", version = "1.6" }
 
 [features]
 async-std = ["async-std-resolver"]
-tokio = ["trust-dns-resolver/tokio-runtime"]
+tokio = ["hickory-resolver/tokio-runtime"]
 # The `tokio-` prefix and feature dependency is just to be explicit,
 # since these features of `trust-dns-resolver` are currently only
 # available for `tokio`.
-tokio-dns-over-rustls = ["tokio", "trust-dns-resolver/dns-over-rustls"]
-tokio-dns-over-https-rustls = ["tokio", "trust-dns-resolver/dns-over-https-rustls"]
```

Comment:
@mxinden I am tempted to remove these features. We don't depend on them and downstream users can easily activate them themselves so I don't really see why we should forward these.

Do you agree?

Reply:
Fine by me.